00-HTML-CSS-basics/css/grid.css

```diff
@@ -0,0 +1,65 @@
+.container {
+    height: 100vh;
+    width: 100vw;
+}
+
+.info {
+    box-sizing: border-box;
+    float: left;
+    font-size: 12px;
+    height: 25vh;
+    margin-left: 2vw;
+    margin-right: 2.5vw;
+    padding-top: 0.5vw;
+    width : 45vw;
+}
+
+.frame {
+    box-sizing: border-box;
+    float:left;
+    margin-left: 2vw;
+    margin-top: 2vw;
+    width: 30vw;
+}
+
+.showcase {
+    box-sizing: border-box;
+    float:left;
+    font-size: 12px;
+    font-weight: bold;
+    height: 25vh;
+    margin-left: 2vw;
+    margin-right: 2.5vw;
+    padding-top: 0.5vw;
+    width : 30vw;
+}
+
+.frame__span {
```

Comment:
Ok, you almost got it! Remember please to take a look at our standards. You used BEM conventions but as the page says and not as we said we were doing. Remember:

```
block
block--element
block--element_modifier
```